test/test_browser.py

```diff
@@ -4313,23 +4222,13 @@ def test_pthread_call_async_on_main_thread(self):
   # global data section of the application memory area.
   @requires_threads
   def test_pthread_global_data_initialization(self):
-    # --memory-init-file mode disabled because WASM=0 does not seem to work with EXPORT_NAME
-    mem_init_modes = [[]]  # ['-sWASM=0', '--memory-init-file', '0'], ['-sWASM=0', '--memory-init-file', '1']]
-    for mem_init_mode in mem_init_modes:
-      print(mem_init_mode)
-      for args in [['-sMODULARIZE', '-sEXPORT_NAME=MyModule', '--shell-file', test_file('shell_that_launches_modularize.html')], ['-O3']]:
-        self.btest_exit('pthread/test_pthread_global_data_initialization.c', args=args + mem_init_mode + ['-pthread', '-sPROXY_TO_PTHREAD', '-sPTHREAD_POOL_SIZE'])
+    for args in [['-sMODULARIZE', '-sEXPORT_NAME=MyModule', '--shell-file', test_file('shell_that_launches_modularize.html')], ['-O3']]:
+      self.btest_exit('pthread/test_pthread_global_data_initialization.c', args=args + ['-pthread', '-sPROXY_TO_PTHREAD', '-sPTHREAD_POOL_SIZE'])
 
   @requires_wasm2js
   @requires_threads
   def test_pthread_global_data_initialization_in_sync_compilation_mode(self):
-    if self.is_wasm64():
-      self.skipTest('wasm2js is not compatible with MEMORY64')
```

Comment:
Why did the wasm64 check vanish?

Reply:
`@requires_wasm2js` already contains this check.  I think this should have been in #21412 perhaps.